Tests/Functional/Components/VatIdValidatorResultTest.php

```diff
@@ -20,20 +20,29 @@ class VatIdValidatorResultTest extends TestCase
 {
     use ContainerTrait;
 
-    public function testSetServiceUnavailable(): void
-    {
-        $container = $this->getContainer();
-        static::assertInstanceOf(Container::class, $container);
+    private const DEFAULT_CONFIG_ALLOW_REGISTER_ON_API_ERROR = false;
 
-        $config = $container->get('config');
-        static::assertInstanceOf(ShopwareConfig::class, $config);
+    /**
+     * @var ShopwareConfig
+     */
+    private $config;
 
-        $config->offsetSet(VatIdConfigReaderInterface::ALLOW_REGISTER_ON_API_ERROR, false);
+    /**
+     * @before
+     */
+    public function setUp(): void
+    {
+        $this->config = $this->getContainer()->get('config');
+    }
+
+    public function testSetServiceUnavailable(): void
+    {
+        $this->config->offsetSet(VatIdConfigReaderInterface::ALLOW_REGISTER_ON_API_ERROR, false);
 
         $validatorResult = $this->createVatIdValidatorResult();
         $validatorResult->setServiceUnavailable();
 
-        $container->reset('config');
+        $this->config->offsetSet(VatIdConfigReaderInterface::ALLOW_REGISTER_ON_API_ERROR, self::DEFAULT_CONFIG_ALLOW_REGISTER_ON_API_ERROR);
```

Comment:
please reset the values to its default

Reply:
the `offsetSet`-method get the `DEFAULT_CONFIG_ALLOW_REGISTER_ON_API_ERROR` - value which is the default 🤔 
phpstan throws an error before because `$container->get('config');` does not have a `reset`-method. (or something like this. I cannot remember)